_targets_eval_postprocessing.R

```diff
@@ -383,6 +395,14 @@ head_to_head_targets <- list(
       dplyr::left_join(table_of_loc_dates_w_ww,
         by = c("location", "forecast_date")
       ) |>
+      dplyr::anti_join(
+        table_of_ww_exclusions |>
```

Comment:
These need to be changed too, no?
```suggestion
        ww_forecast_date_locs_to_excl |>
```

Reply:
Oops my bad good catch